plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/BlueOceanHook.java

```diff
@@ -26,7 +26,7 @@
     public static final List<String> BO_PLUGINS = Arrays.asList("blueocean", "blueocean-commons",
             "blueocean-config", "blueocean-dashboard", "blueocean-events", "blueocean-git-pipeline", "blueocean-github-pipeline",
             "blueocean-i18n", "blueocean-jwt", "blueocean-personalization", "blueocean-pipeline-api-impl", "blueocean-rest",
-            "blueocean-rest-impl", "blueocean-web", "blueocean-pipeline-scm-api");
+            "blueocean-rest-impl", "blueocean-web", "blueocean-pipeline-scm-api", "blueocean-pipeline-editor");
```

Comment:
On 1.3 we have new one blueocean-jira.

Reply:
@vivek Yes, I have added it already, we need to find a better way to determine if a plugin belongs to the BO agreggator project or we are going to end making this kind of fixes continously, but that can be done in a follow up PR